components/molecules/ContributorProfileInfo/contributor-profile-info.tsx

```diff
@@ -54,7 +54,7 @@ const ContributorProfileInfo = ({
           {isConnected && <Badge text="beta" />}
         </div>
         <div className="flex items-center text-sm gap-3">
-          <span className="text-light-slate-11 text-sm">{`@${githubName}`}</span>
+          <a className="text-light-slate-11 text-sm" href={`https://github.com/${githubName}`}>{`@${githubName}`}</a>
```

Comment:
If we have the user back to their profile, then it would not make sense to add a link here. 
https://deploy-preview-1449--oss-insights.netlify.app/user/a0m0rajab/highlights

Reply:
I agree with @babblebey on this. We should ignore making a link to Github from the user's profile for now. we can revisit this in the future and maybe add some design help from @isabensusan.

Reply:
@bdougie already agreed on that in the previous issue though,

Reply:
I agreed on the PR, code review is part of the process. 

I also stand by my previous feedback in the issue. It is not clear what the goal is in this PR. The title seems like this PR is two features in on PR.

This PR links 3 different issues. I don't know where to begin on the feedback or review. 

I'd suggest closing this PR and opening separate PRs.

Reply:
Re: 3 different issues, 

I thought this is an accepted pattern since I saw it before in the AI repo 
![image](https://github.com/open-sauced/insights/assets/18273833/a4c1d19f-17c6-417f-9ac6-16ac2bda15e9)

https://github.com/open-sauced/ai/pull/163

Re: new PR. 
Opened three different PRs.

Reply:
> I thought this is an accepted pattern since I saw it before in the AI repo

This is a repo used by close to 4k users.

We can close multiple issues when it makes sense, but your changes were completely different and your title and descriptions were not cohesive.

There are a lot more 👀 here than the AI repo.

Please consider the context

Reply:
Thank you for giving me information about the context! Was looking at all repos from the same perspective 😄